### ISSUE TITLE
Success text missing ARIA role

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-modules/Collecting#94]
Collecting form submissions prompt a success message, either automatically generated or user customized. `role="status"` should be added to the container of these messages in order to indicate this status update to screen reader users.
